presto-hive/src/test/java/com/facebook/presto/hive/TestCoercingFilters.java

```diff
@@ -70,4 +79,41 @@ public void testVarcharToInteger()
 
         assertFalse(coercingFilter.testNull());
     }
+
+    private final TypeManager typeManager = MetadataManager.createTestMetadataManager().getFunctionAndTypeManager();
```

Comment:
In Presto, the data fields are defined at the beginning of the class. Could you please move this to line 40?